security/mk-cacerts.sh

```diff
@@ -54,7 +67,7 @@ fi
 
 # Convert Mozilla's list of certificates into a PEM file. The -n switch makes
 # it use the local certdata.txt in this folder.
-certNum=$(./mk-ca-bundle.pl -v -n ca-bundle.crt)
+certNum=$(${SCRIPT_DIR}/mk-ca-bundle.pl -v -n ca-bundle.crt)
```

Comment:
Curious - how are you invoking this in a way where the original logic (without the explicit `SCRIPT_DIR`) didn't work?

Reply:
That commit most likely is not needed.  The whole message was "./mk-ca-bundle.pl Couldn't open file: " - I had misinterpreted it as  that `./mk-ca-bundle.pl can not be opened` as I overlooked the tailing dangling `:` , and so tried to replace the `.` by full path. Unluckily committed it and then forge to remove it. Sorry for confusion

Reply:
Jut note whether to keep that or remove that. I do not h ave strong preference.

Reply:
My preference would be to remove it purely because of all the extra lines of code that were added to set `SCRIPT_DIR` :-)

(Original commit for historic reference in case we remove it and want to find it in the future: https://github.com/adoptium/temurin-build/pull/3796/commits/55f1195eec2b90a16bcab15795fd6f1bbb8d9921)

Reply:
Sure. Will be done!